src/components/date/navbar/navbar.js

```diff
@@ -9,20 +10,20 @@ const Navbar = ({
 }) => {
   return (
     <div className={ className }>
-      <button
+      <StyledButton
+        className='DayPicker-NavButton--prev'
```

Comment:
do we need that class?

Reply:
It is used in unit test to find button and simulate click event.